my-code/src/Components/SearchMovieInput/styles.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import { theme } from '../../GlobalStyles/Theme';
+
+export const SearchMoviesInputContainer = styled.div`
+    width: 100%;
+    height: 44px;
```

Comment:
Since this contains text, you should also consider if using px here is worthwhile (I exaggerated and put in 120px as the root font size):
![CleanShot 2022-07-13 at 16 33 56](https://user-images.githubusercontent.com/26071571/178773316-80b3579c-408b-45b6-a2e1-18eea6f05e7f.png)